components/AppShell.vue

```diff
@@ -6,128 +6,64 @@
     >
       <Dialog
         as="div"
-        class="relative z-40 md:hidden"
-        @close="sidebarOpen = false"
+        class="relative md:hidden"
       >
-        <TransitionChild
-          as="template"
-          enter="transition-opacity ease-linear duration-300"
-          enter-from="opacity-0"
-          enter-to="opacity-100"
-          leave="transition-opacity ease-linear duration-300"
-          leave-from="opacity-100"
-          leave-to="opacity-0"
-        >
-          <div class="fixed inset-0 bg-gray-600 bg-opacity-75" />
-        </TransitionChild>
-
-        <div class="fixed inset-0 z-40 flex">
+        <div class="fixed inset-x-0 inset-y-16 flex min-h-full">
           <TransitionChild
             as="template"
             enter="transition ease-in-out duration-300 transform"
-            enter-from="-translate-x-full"
-            enter-to="translate-x-0"
+            enter-from="translate-y-full"
+            enter-to="translate-y-0"
             leave="transition ease-in-out duration-300 transform"
-            leave-from="translate-x-0"
-            leave-to="-translate-x-full"
+            leave-from="translate-y-0"
+            leave-to="translate-y-full"
```

Comment:
Menu transitions from the bottom of the page on mobile now. 

I tried from the top but it means the reader had to wait longer for menu items to be visible

Reply:
I've updated the menu to transition from below the navbar after design feedback 👍 

Have also doubled the speed of the transition to make it feel snappy ⚡